pkgs/development/python-modules/tinytuya/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cryptography
+, requests
+, colorama
+}:
+
+buildPythonPackage rec {
+  pname = "tinytuya";
+  version = "1.13.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

and then `build-system = [ setuptools ];`. See https://nixos.org/manual/nixpkgs/stable/#python